kitsune/questions/templates/questions/question_details.html

```diff
@@ -118,7 +118,9 @@ <h2 class="summary" itemprop="headline">{{ question.title }}</h2>
         </div>
         <div class="asked-on">
           {{ _('Posted') }}<br>
-          <time itemprop="dateCreated" datetime="{{ question.created }}">{{ datetimeformat(question.created) }}</time>
+          <span itemprop="dateCreated">
+            {{ datetimeformat(question.created) }}
+          </span>
```

Comment:
I don't think this change will do what we want, though I'm not sure. The itemprop attribute is used as part of a microdata schema. I think you may need to modify `datetimeformat` to add extra parameters like itemprop.

@rehandalal You implemented the microdata stuff, right? Will it be fine to have the itemprop attribute on a surrounding `<span>` instead of on the `<time>` element that `datetimeformat` generates?